packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -143,7 +151,7 @@ const getProps = ({
   contrast: NonNullable<BaseLinkProps['contrast']>;
   isVisited: boolean;
   target: BaseLinkProps['target'];
-  hasIcon: boolean;
```

Comment:
was this not used anywhere? don't see any change in any component regarding this 🤔

Reply:
This was used for conditionally adjusting icon sizes based on whether:
1. icon only link was used
2. link with icon and text was used

Now that we're going to have a consistent icon size, simplified the check and removed this flag.